packages/material-ui/src/layout.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Paper from 'material-ui/Paper';
+
+const Layout = ({ children }) => (
+  <Paper style={{ margin: 'auto', maxWidth: 400, marginTop: 50, padding: 20 }}>
```

Comment:
- We can add a LayoutStyle prop
- We can add a Layout prop that can overwrite the package Layout with a user Component